### PR TITLE
Updating TP statuses for my teams

### PR DESCRIPTION
**Pull request by bergerhoffer**
Preview: https://deploy-preview-31584--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-8-release-notes.html#ocp-4-8-technology-preview

### CODE REVIEW

File: release_notes/ocp-4-8-release-notes.adoc [hunk 1]
```diff
@@ -351,12 +351,12 @@ In the table below, features are marked with the following statuses:
 |Scheduler profiles
 |-
 |TP
-|
+|TP
```
Comment (bergerhoffer) [inline release_notes/ocp-4-8-release-notes.adoc:354]: @damemi Can you confirm that scheduler profiles are staying tech preview for 4.8?

File: release_notes/ocp-4-8-release-notes.adoc [hunk 2]
```diff
@@ -351,12 +351,12 @@ In the table below, features are marked with the following statuses:
...
 
 |Non-preempting priority classes
 |-
 |TP
-|
+|TP
```
Comment (bergerhoffer) [inline release_notes/ocp-4-8-release-notes.adoc:359]: @ingvagabund Can you confirm that non-preempting priority classes are staying tech preview for 4.8?

**Comment by ahardin-rh** [comment]
LGTM!

File: release_notes/ocp-4-8-release-notes.adoc (hunk 1 above)
Comment (damemi) [reply release_notes/ocp-4-8-release-notes.adoc:354]: @soltysh did these officially get moved to 4.9?

File: release_notes/ocp-4-8-release-notes.adoc (hunk 2 above)
Comment (ingvagabund) [reply release_notes/ocp-4-8-release-notes.adoc:359]: I don't recall there was any change in this respect. @soltysh the same question here as well. Is there a top level card which tracks which features is planned to be GA'ed?

File: release_notes/ocp-4-8-release-notes.adoc (hunk 1 above)
Comment (soltysh) [reply release_notes/ocp-4-8-release-notes.adoc:354]: Yes, GA in 4.9, as discussed in slack.

**Review by soltysh — APPROVED** [review]
/lgtm

**Pull request merged**